### PR TITLE
Async: Add description parameter of assert.async()

### PR DESCRIPTION
**Pull request by mixed**
Accept call count of callback of assert.async()
Ref jquery/qunit#863

### CODE REVIEW

File: entries/async.xml [hunk 1]
```diff
@@ -2,11 +2,16 @@
 <?xml-stylesheet type="text/xsl" href="../entries2html.xsl" ?>
 <entry type="method" name="async">
   <title>async()</title>
+  <signature>
+    <argument name="acceptCallCount" type="Number" default="1">
```
Comment (leobalter) [inline entries/async.xml]: would you include an `optional="true"` attribute here?

Comment (mixed) [inline entries/async.xml]: @leobalter 
OK. I will update it.

File: entries/async.xml [hunk 2]
```diff
@@ -2,11 +2,16 @@
 <?xml-stylesheet type="text/xsl" href="../entries2html.xsl" ?>
 <entry type="method" name="async">
   <title>async()</title>
+  <signature>
+    <argument name="acceptCallCount" type="Number" optional="true" default="1">
+      <desc>Number accept a call count of callback of assert.async().</desc>
```
Comment (jzaefferer) [inline entries/async.xml]: How about "Number of expected callbacks before the test is done"

File: entries/async.xml [hunk 3]
```diff
@@ -2,11 +2,16 @@
...
+    </argument>
+  </signature>
   <desc>
     Instruct QUnit to wait for an asynchronous operation.
   </desc>
   <longdesc>
-    <p>When your test has any asynchronous exit points, call <code>assert.async()</code> to get a unique resolution callback for each async operation. The callback returned from <code>assert.async()</code> will throw an Error if is invoked more than once.</p>
+    <p>When your test has any asynchronous exit points, call <code>assert.async()</code> to get a unique resolution callback for each async operation. The callback returned from <code>assert.async()</code> will throw an Error if is invoked more than accept call count.</p>
```
Comment (jzaefferer) [inline entries/async.xml]: How about "The callback returned from <code>assert.async()</code> will throw an Error if it is invoked more than once (or more often than the accepted call count, if provided)."

File: entries/async.xml [hunk 4]
```diff
@@ -41,7 +46,30 @@ QUnit.test( "two async calls", function( assert ) {
 });
 ]]></code>
   </example>
+    <example height="250">
+  <desc>Establish an accept call count. The <code>done()</code> callback can be called at accpet call count.</desc>
```
Comment (jzaefferer) [inline entries/async.xml]: How about "Set up an async test three exit points. Each <code>done()</code> call adds up to the <code>acceptCallCount</code>. After three calls, the test is done."

File: entries/async.xml (hunk 2 above)
Comment (kswedberg) [inline entries/async.xml]: :+1:

**Comment by mixed** [comment]
@jzaefferer 
I updated it. Thank you for your advice.

**Comment by leobalter** [comment]
Thanks, @mixed!

**Comment by jzaefferer** [comment]
:+1:

**Pull request closed without merging**